doc/nest_by_example/NEST by Example - An Introduction to the Neural Simulation Tool NEST Version 2.12.0.ipynb

```diff
@@ -0,0 +1,2437 @@
+    "nodes intact and will create a second instance for each node. To start\n",
+    "a new NEST session without leaving Python, you can call\n",
+    "`nest.ResetKernel()`. This function will erase the existing\n",
+    "network so that you can start from scratch.\n",
+    "\n",
+    "# Example 1: A sparsely connected recurrent network\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we discuss a model of activity dynamics in a local cortical\n",
+    "network proposed by <cite data-cite=\"Brunel00\">Brunel (2000)</cite>. We only describe those parts of\n",
+    "the model which are necessary to understand its NEST\n",
+    "implementation. Please refer to the original paper for further details.\n",
+    "\n",
+    "The local cortical network consists of two neuron populations: a\n",
+    "population of $N_E$ excitatory neurons and a population of $N_I$\n",
+    "inhibitory neurons. To mimic the cortical ratio of 80% excitatory neurons\n",
+    "and 20% inhibitory neurons, we assume that $N_E=$ 8000 and $N_I=$ 2000. Thus,\n",
+    "our local network has a total of 10,000 neurons.\n",
+    "\n",
+    "For both the excitatory and the inhibitory population, we use the same\n",
+    "integrate-and-fire neuron model with current-based synapses. Incoming\n",
+    "excitatory and inhibitory spikes displace the membrane potential $V_m$\n",
+    "by $J_{E}$ and $J_I$, respectively. If $V_m$ reaches the threshold\n",
+    "value $V_{\\text{th}}$, the membrane potential is reset to $V_{\\text{reset}}$,\n",
+    "a spike is sent with delay $D=$ 1.5 ms to all post-synaptic\n",
+    "neurons, and the neuron remains refractory for $\\tau_{\\text{rp}}=$ 2.0 ms.\n",
+    "\n",
+    "The neurons are mutually connected with a probability of\n",
+    "10%. Specifically, each neuron receives input from $C_{E}=$ 0.1 $\\cdot N_{E}$ excitatory and $C_I=\\cdot N_{I}$ inhibitory neurons (see figure below). The inhibitory synaptic weights\n",
+    "$J_I$ are chosen with respect to the excitatory synaptic weights $J_E$\n",
+    "such that \n",
+    "$J_I = -g \\cdot J_E$, \n",
+    "with $g=$ 5.0 in this example.\n",
+    "\n",
+    "\n",
+    "\n",
+    "<table class=\"image\">\n",
+    "<caption align=\"bottom\">The network consists of three populations: $N_E$ excitatory neurons (circle labeled E), $N_I$ inhibitory neurons (circle labeled I), and a population of identical, independent Poisson processes (PGs) representing activity from outside the network. Arrows represent connections between the network nodes. Triangular arrow-heads represent excitatory and round arrow-heads represent inhibitory connections. The numbers at the start and end of each arrow indicate the multiplicity of the connection.</caption>\n",
+    "<tr><td><img src=\"figures/brunel_detailed_external_single2.jpg\" alt=\"Brunel detailed network\" style=\"width: 500px;\"/></td></tr>\n",
```

Comment:
I would maybe remove the "width" specification. Or is there a way to specify that you want a width of at most 500px?